oak-lucene/src/test/java/org/apache/jackrabbit/oak/plugins/index/lucene/dynamicBoost/DynamicBoostTest.java

```diff
@@ -128,6 +128,20 @@ protected ContentRepository createRepository() {
                         "]", log);
     }
 
+    @Test public void withDynamicBoostLite() throws Exception {
+        NodeTypeRegistry.register(root, toInputStream(ASSET_NODE_TYPE), "test nodeType");
+        Tree props = createIndex("dam:Asset", true);
+        Tree pt = createNodeWithType(props, "predictedTags", UNSTRUCTURED);
+        pt.setProperty("name", "jcr:content/metadata/predictedTags/.*");
+        pt.setProperty("isRegexp", true);
+        pt.setProperty("dynamicBoost", true);
```

Comment:
`dynamicBoostLite ` ?

Reply:
no, here `dynamicBoost` is correct. `dynamicBoostLite` is something at index definition level.